saml-handler/src/main/java/org/apache/sling/auth/saml2/AuthenticationHandlerSAML2Config.java

```diff
@@ -15,6 +15,8 @@
  * KIND, either express or implied.  See the License for the
  * specific language governing permissions and limitations
  * under the License.
+ *
+ * Contributed by Cris Rockwell and the Regents of the University of Michigan.
```

Comment:
I don't believe this belongs in the license header. Attribution is done using the git author field.

Reply:
Ok. I can remove it.